src/main/java/games/strategy/util/IllegalCharacterRemover.java

```diff
@@ -12,28 +12,28 @@
    * Designed to remove / \b \n \r \t \0 \f ` ? * \ < > | " ' : . , ^ [ ] = + ;
    */
   public static String removeIllegalCharacter(final String text) {
-    final StringBuilder rVal = new StringBuilder();
+    final StringBuilder sb = new StringBuilder();
     for (int i = 0; i < text.length(); ++i) {
       if (!isIllegalFileNameChar(text.charAt(i))) {
-        rVal.append(text.charAt(i));
+        sb.append(text.charAt(i));
       }
     }
-    return rVal.toString();
+    return sb.toString();
   }
 
   /**
    * Designed to replace / \b \n \r \t \0 \f ` ? * \ < > | " ' : . , ^ [ ] = + ;
    */
   public static String replaceIllegalCharacter(final String text, final char replacement) {
```

Comment:
Seems like there should be an apache lib or guava that could do this as a one liner.

Reply:
Yes.  Guava's [`CharMatcher`](http://google.github.io/guava/releases/19.0/api/docs/com/google/common/base/CharMatcher.html) looks like it can be used to both remove and replace in a single statement.  I'll write some characterization tests and submit a PR if it works.